pkg/image-manager/mocks/common_api_client.go

```diff
@@ -0,0 +1,2046 @@
+// Code generated by mockery v1.0.0. DO NOT EDIT.
```

Comment:
there is already a mock for docker client in `pkg/functions/docker/mocks`, maybe we could move mocks of external services into a top-level mocks directory and re-use them

Reply:
Yeah, we could. I will move them.